package.json

```diff
@@ -9,13 +9,14 @@
     "node": ">=12.13"
   },
   "scripts": {
-    "docs": "yarn --cwd packages/docs start",
+    "docs": "yarn --cwd packages/docs start --no-clear-screen",
```

Comment:
simple curiosity: what's this do?

Reply:
by default when vuepress starts it clears the terminal output, this has led to times where I've started up the server, and it appears to have been successful but wont work. In those instances, it had wiped away compilation errors - so I added this flag.